src/AltinnCore/Common/Services/Implementation/ExecutionSIContainer.cs

```diff
@@ -0,0 +1,272 @@
+﻿
```

Comment:
I can see that this file is similar to ExecutionSILocalDev and that its in need of some fixing since there are some unecessary linebreaks here and there, some unused code and also some outcommented code. I suggest that this eiter be fixed here or create an issue with this techincal dept.

Reply:
We will enable stylecop in next sprint